### DISCUSSION TITLE
Pull Request says nothing to compare.

### DISCUSSION THREAD

**Discussion by knguyen111601** [Pantheon-Labs/Software-Engineer-I#15] [Get Help] [answered]
Hi, my name is Kenny Nguyen and I am getting this issue when I try to make a pull request between my forked repo and the original Pantheon repo. It says that there's nothing to compare as they are entirely different commit histories. Upon looking it up online, I found out it may be because of how I initialized the repository. The solution seems to be to clone it into a new repo and try again but I didn't want to lose my commit history.  Thank you and sorry for the trouble

**Comment by SpencerKaiser** [comment, marked as answer]
Hi @knguyen111601 - if you created a new repo instead of forking/cloning this one, your commit history won't share a base and git won't know how to merge it.

You should be able to merge the two; try [this post on Stack overflow](https://stackoverflow.com/a/10548919/3739332)

**Reply by knguyen111601** [reply]
Hello @SpencerKaiser, I just did what was suggested in the Slack and it worked. I submitted the pull request just now. Thank you for the help!
